pkg/corerp/frontend/controller/applications/testdata/graph-app-httproute-out.json

```diff
@@ -0,0 +1,47 @@
+[
+    {
+        "connections": [
+            {
+                "direction": "Outbound",
+                "id": "/planes/radius/local/resourcegroups/default/providers/Applications.Core/httpRoutes/sql-rte"
+            }
+        ],
+        "id": "/planes/radius/local/resourcegroups/default/providers/Applications.Core/containers/sql-ctnr",
+        "name": "sql-ctnr",
+        "outputResources": [],
+        "provisioningState": "Succeeded",
+        "type": "Applications.Core/containers"
+    },
+    {
+        "id": "/planes/radius/local/resourcegroups/default/providers/Applications.Datastores/sqlDatabases/sql-db",
+        "name": "sql-db",
+        "provisioningState": "Succeeded",
+        "type": "Applications.Datastores/sqlDatabases"
+    },
+    {
+        "connections": [
+            {
+                "direction": "Inbound",
+                "id": "/planes/radius/local/resourcegroups/default/providers/Applications.Core/containers/sql-ctnr"
+            }
+        ],
+        "id": "/planes/radius/local/resourcegroups/default/providers/Applications.Core/httpRoutes/sql-rte",
+        "name": "sql-rte",
+        "outputResources": [],
+        "provisioningState": "Succeeded",
+        "type": "Applications.Core/httpRoutes"
+    },
+    {
+        "connections": [
+            {
+                "direction": "Inbound",
```

Comment:
isn't this outbound connection? is this because `sql-app-ctnr` gets the data from `sql-db`.

Reply:
I think there's a bug where the data is incorrect. Take a look at the CLI `rad app graph` for a deployed application. 

If you use the tutorial app then there should be a connection like `demo -> db`. If the arrow is going the opposite way then it was a regression. 

/cc @nithyatsu

Reply:
I fixed the bug. Please see the new testdata.

https://github.com/radius-project/radius/blob/fc4594ecdf5f2a9fc818739e925f10506c6cf980/pkg/corerp/frontend/controller/applications/graph_util.go#L496